### DISCUSSION TITLE
Welcome to Bonsai Discussions!

### DISCUSSION THREAD

**Discussion by glopesdev** [bonsai-rx/bonsai#864] [Announcements]
## 👋 Welcome!

We are introducing Discussions as an improved channel for connecting with members of the Bonsai community. We were previously using the [Bonsai Users](https://groups.google.com/g/bonsai-users) group for this purpose, but there are a number of issues with Google groups that make it harder to scale to our increasingly large community, so we are considering alternatives. If you have an active question that is still in the Google group, please consider reproducing it here to help us try out this alternative.

We hope that you:

  * Search the forum for similar questions before posting a new one (keywords, e.g. tracking, or node names, e.g. `SelectMany`).
  * Use descriptive and relevant titles. We suggest using the question itself as a title.
  * Include a picture of the workflow. You can copy an image of the workflow using `Ctrl+Shift+C` in the editor, and then paste it directly in the body of the discussion.
  * Attach a ZIP file including your workflow, together with other supporting media such as images, videos, data, or other files that can help illustrate the issue. Be mindful of sharing sensitive content.
  * Help us answer questions and join the conversations.
  * Welcome others and uphold our community [Code of Conduct](https://bonsai-rx.org/code-of-conduct/). Remember this is a community we build together 💪.

Feel free to comment below with an introduction of yourself and tell us about what you do with Bonsai.

**Comment by taunsquared** [comment]
Hi everyone, my name is [Danbee Kim](https://github.com/taunsquared) and I'm the Research and Outreach Scientist at [NeuroGEARS](https://github.com/neurogears). I mainly work on community management for Bonsai, which includes organizing online learning resources and other forms of documentation for Bonsai. I'm hoping that by helping document and teach Bonsai, I can also improve my own fluency with this programming language!

**Comment by bruno-f-cruz** [comment]
Hi! My name is Bruno Cruz. I have been using Bonsai for some years, as a PhD Neuroscience student, to prototype solutions to everyday problems that I come across in the lab. 

I have recently joined NeuroGEARS as a Research Scientist where I continue to use Bonsai to develop technical solutions. 
I look forward to keep on helping people out with their questions, and further helping this community grow!

See you around 😄
